pkg/controllers/analytics.go

```diff
@@ -30,3 +30,9 @@ func CaptureCommand(wg *sync.WaitGroup, command string) {
 	command = strings.ReplaceAll(command, " ", ".")
 	http.CaptureCommand(command)
 }
+
+func CaptureEvent(wg *sync.WaitGroup, event string) {
+	defer wg.Done()
+
+	http.CaptureEvent(event)
```

Comment:
nit: To be very explicit that we're ignoring these returned values (and swallowing errors), it might be nice to assign the returned values to underscore:

```diff
diff --git a/pkg/controllers/analytics.go b/pkg/controllers/analytics.go
index 6d07189..4637176 100644
--- a/pkg/controllers/analytics.go
+++ b/pkg/controllers/analytics.go
@@ -28,11 +28,11 @@ func CaptureCommand(wg *sync.WaitGroup, command string) {
 	defer wg.Done()
 
 	command = strings.ReplaceAll(command, " ", ".")
-	http.CaptureCommand(command)
+	_, _ = http.CaptureCommand(command)
 }
 
 func CaptureEvent(wg *sync.WaitGroup, event string) {
 	defer wg.Done()
 
-	http.CaptureEvent(event)
+	_, _ = http.CaptureEvent(event)
 }


```

Reply:
Ended up debug logging the error message